16:32:02,505 INFO  [root] --------------- session started -----------------------
16:32:02,505 INFO  [root] logging facility to "example.log"
16:32:02,505 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:32:10,894 INFO  [root] --------------- session started -----------------------
16:32:10,895 INFO  [root] logging facility to "example.log"
16:32:10,895 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:32:32,368 INFO  [root] --------------- session started -----------------------
16:32:32,368 INFO  [root] logging facility to "example.log"
16:32:32,368 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:33:01,912 INFO  [root] --------------- session started -----------------------
16:33:01,912 INFO  [root] logging facility to "example.log"
16:33:01,912 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:33:35,531 INFO  [root] --------------- session started -----------------------
16:33:35,531 INFO  [root] logging facility to "example.log"
16:33:35,532 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:34:08,139 INFO  [root] --------------- session started -----------------------
16:34:08,140 INFO  [root] logging facility to "example.log"
16:34:08,140 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:34:22,668 INFO  [root] --------------- session started -----------------------
16:34:22,668 INFO  [root] logging facility to "example.log"
16:34:22,669 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
16:34:33,237 INFO  [root] --------------- session started -----------------------
16:34:33,237 INFO  [root] logging facility to "example.log"
16:34:33,237 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:17:08,281 INFO  [root] --------------- session started -----------------------
17:17:08,281 INFO  [root] logging facility to "example.log"
17:17:08,282 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:18:06,510 INFO  [root] --------------- session started -----------------------
17:18:06,510 INFO  [root] logging facility to "example.log"
17:18:06,510 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:18:24,204 WARNI [root] 404 GET / (127.0.0.1) 0.48ms
17:18:26,920 WARNI [root] 404 GET /ui/logout (127.0.0.1) 0.36ms
17:18:29,214 ERROR [root] Uncaught exception GET /ui (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8080', method='GET', uri='/ui', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'en-us', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'auth_token=eyJhaSI6IDIsICJwaSI6IDIsICJhYiI6ICJhcHAxIiwgInJiIjogInJ0MSIsICJwdCI6ICJ1In0=|1330080056|5df31548f34e0e71071bb5b84c8d8e83aa164c6a; realm=xfirstx; user_locale=ZW5fVVM=|1329910444|13a77a78d97b2531fdc4de8c0f90fbb10eec2a11', 'Host': '127.0.0.1:8080', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/534.52.7 (KHTML, like Gecko) Version/5.1.2 Safari/534.52.7'})
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 861, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "/Users/Stan/Sources/tornado-extjs-glue/example/ui.py", line 22, in get
    supported_locales = locales )
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 419, in render
    html = self.render_string(template_name, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 516, in render_string
    t = RequestHandler._templates[template_path].load(template_name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/template.py", line 192, in load
    f = open(path, "r")
IOError: [Errno 2] No such file or directory: '/Users/Stan/Sources/tornado-extjs-glue/example/templates/index.html'
17:18:29,240 ERROR [root] 500 GET /ui (127.0.0.1) 25.85ms
17:18:34,890 WARNI [root] 404 GET /tag (127.0.0.1) 0.32ms
17:18:41,543 ERROR [root] Uncaught exception GET /api/tag (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8080', method='GET', uri='/api/tag', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'en-us', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'auth_token=eyJhaSI6IDIsICJwaSI6IDIsICJhYiI6ICJhcHAxIiwgInJiIjogInJ0MSIsICJwdCI6ICJ1In0=|1330080056|5df31548f34e0e71071bb5b84c8d8e83aa164c6a; realm=xfirstx; user_locale=ZW5fVVM=|1329910444|13a77a78d97b2531fdc4de8c0f90fbb10eec2a11', 'Host': '127.0.0.1:8080', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/534.52.7 (KHTML, like Gecko) Version/5.1.2 Safari/534.52.7'})
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 861, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "/Users/Stan/Sources/tornado-extjs-glue/teg/controller.py", line 51, in wrapper
    answer = method(self, *args, **kwargs)
  File "/Users/Stan/Sources/tornado-extjs-glue/example/api.py", line 34, in get
    query = self.apply_filering(model.Tag, query)
AttributeError: 'Tag' object has no attribute 'apply_filering'
17:18:41,544 ERROR [root] 500 GET /api/tag (127.0.0.1) 6.31ms
17:19:04,226 INFO  [root] --------------- session started -----------------------
17:19:04,226 INFO  [root] logging facility to "example.log"
17:19:04,226 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:19:06,602 DEBUG [teg.controller] no filter arguments
17:19:06,602 DEBUG [teg.controller] no sort arguments
17:19:06,603 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:19:06,604 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:19:06,604 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:19:06,605 ERROR [root] Uncaught exception GET /api/tag (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8080', method='GET', uri='/api/tag', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Accept-Language': 'en-us', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:8080', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/534.52.7 (KHTML, like Gecko) Version/5.1.2 Safari/534.52.7', 'Connection': 'keep-alive', 'Cookie': 'auth_token=eyJhaSI6IDIsICJwaSI6IDIsICJhYiI6ICJhcHAxIiwgInJiIjogInJ0MSIsICJwdCI6ICJ1In0=|1330080056|5df31548f34e0e71071bb5b84c8d8e83aa164c6a; realm=xfirstx; user_locale=ZW5fVVM=|1329910444|13a77a78d97b2531fdc4de8c0f90fbb10eec2a11', 'Cache-Control': 'max-age=0'})
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 861, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "/Users/Stan/Sources/tornado-extjs-glue/teg/controller.py", line 51, in wrapper
    answer = method(self, *args, **kwargs)
  File "/Users/Stan/Sources/tornado-extjs-glue/example/api.py", line 36, in get
    return self.generic_get(query, 'tags', oid)
  File "/Users/Stan/Sources/tornado-extjs-glue/teg/controller.py", line 131, in generic_get
    if obj == None: raise teg.exc.ObjectNotFound().object_id(oid)
AttributeError: 'module' object has no attribute 'ObjectNotFound'
17:19:06,607 ERROR [root] 500 GET /api/tag (127.0.0.1) 9.55ms
17:19:45,718 INFO  [root] --------------- session started -----------------------
17:19:45,718 INFO  [root] logging facility to "example.log"
17:19:45,718 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:19:48,360 DEBUG [teg.controller] no filter arguments
17:19:48,360 DEBUG [teg.controller] no sort arguments
17:19:48,361 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:19:48,363 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:19:48,363 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:19:48,364 ERROR [root] Uncaught exception, closing connection.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:19:48,394 ERROR [root] Error in connection callback
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 266, in _handle_events
    self.no_keep_alive, self.xheaders)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 292, in __init__
    self.stream.read_until("\r\n\r\n", self._header_callback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 133, in read_until
    if self._read_from_buffer():
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 325, in _read_from_buffer
    self._consume(loc + delimiter_len))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:19:48,402 DEBUG [teg.controller] no filter arguments
17:19:48,402 DEBUG [teg.controller] no sort arguments
17:19:48,404 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:19:48,404 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:19:48,405 ERROR [root] Uncaught exception, closing connection.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:19:48,405 ERROR [root] Error in connection callback
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 266, in _handle_events
    self.no_keep_alive, self.xheaders)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 292, in __init__
    self.stream.read_until("\r\n\r\n", self._header_callback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 133, in read_until
    if self._read_from_buffer():
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 325, in _read_from_buffer
    self._consume(loc + delimiter_len))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:19:48,408 DEBUG [teg.controller] no filter arguments
17:19:48,408 DEBUG [teg.controller] no sort arguments
17:19:48,409 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:19:48,409 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:19:48,409 ERROR [root] Uncaught exception, closing connection.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:19:48,410 ERROR [root] Error in connection callback
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 266, in _handle_events
    self.no_keep_alive, self.xheaders)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 292, in __init__
    self.stream.read_until("\r\n\r\n", self._header_callback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 133, in read_until
    if self._read_from_buffer():
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 325, in _read_from_buffer
    self._consume(loc + delimiter_len))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/iostream.py", line 230, in _run_callback
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 173, in wrapped
    callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/httpserver.py", line 353, in _on_headers
    self.request_callback(self._request)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 1204, in __call__
    handler._execute(transforms, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 863, in _execute
    self.finish()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/stack_context.py", line 120, in __exit__
    return self.exception_handler(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 844, in _stack_context_handle_exception
    self._handle_request_exception(value)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/tornado-1.2.1-py2.7.egg/tornado/web.py", line 889, in _handle_request_exception
    if e.log_message:
AttributeError: 'NotFound' object has no attribute 'log_message'
17:20:05,741 INFO  [root] --------------- session started -----------------------
17:20:05,741 INFO  [root] logging facility to "example.log"
17:20:05,741 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:20:07,894 DEBUG [teg.controller] no filter arguments
17:20:07,894 DEBUG [teg.controller] no sort arguments
17:20:07,895 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:20:07,897 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:20:07,897 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:20:07,899 WARNI [root] 404 GET /api/tag (127.0.0.1) 9.60ms
17:20:41,992 DEBUG [teg.controller] no filter arguments
17:20:41,992 DEBUG [teg.controller] no sort arguments
17:20:41,993 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:20:41,994 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:20:41,995 WARNI [root] 404 GET /api/tag/1 (127.0.0.1) 3.39ms
17:21:00,362 INFO  [root] --------------- session started -----------------------
17:21:00,362 INFO  [root] logging facility to "example.log"
17:21:00,362 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:21:02,462 DEBUG [teg.controller] no filter arguments
17:21:02,463 DEBUG [teg.controller] no sort arguments
17:21:02,465 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:21:02,466 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:21:02,466 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:21:02,468 WARNI [root] 404 GET /api/tag/1 (127.0.0.1) 11.27ms
17:22:12,209 INFO  [root] --------------- session started -----------------------
17:22:12,209 INFO  [root] logging facility to "example.log"
17:22:12,209 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:22:14,316 DEBUG [teg.controller] no filter arguments
17:22:14,316 DEBUG [teg.controller] no sort arguments
17:22:14,316 DEBUG [teg.controller] get single object with id [tags]
17:22:14,317 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:22:14,318 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:22:14,318 INFO  [sqlalchemy.engine.base.Engine] ('tags', 1, 0)
17:22:14,320 WARNI [root] 404 GET /api/tag/1 (127.0.0.1) 8.94ms
17:23:01,965 INFO  [root] --------------- session started -----------------------
17:23:01,965 INFO  [root] logging facility to "example.log"
17:23:01,965 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:23:04,074 DEBUG [teg.controller] no filter arguments
17:23:04,074 DEBUG [teg.controller] no sort arguments
17:23:04,075 DEBUG [teg.controller] get single object with id [1]
17:23:04,076 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:23:04,078 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag 
WHERE tag.id = ?
 LIMIT ? OFFSET ?
17:23:04,078 INFO  [sqlalchemy.engine.base.Engine] ('1', 1, 0)
17:23:04,079 WARNI [root] 404 GET /api/tag/1 (127.0.0.1) 11.23ms
17:23:06,901 DEBUG [teg.controller] no filter arguments
17:23:06,902 DEBUG [teg.controller] no sort arguments
17:23:06,902 DEBUG [teg.controller] arguments: {}
17:23:06,905 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag) AS anon_1
17:23:06,905 INFO  [sqlalchemy.engine.base.Engine] ()
17:23:06,906 DEBUG [teg.controller] get page 1, offset 0, limit 20
17:23:06,906 DEBUG [teg.controller] (SELECT tag.id, tag.name 
FROM tag
 LIMIT :param_1)
17:23:06,908 INFO  [sqlalchemy.engine.base.Engine] SELECT tag.id AS tag_id, tag.name AS tag_name 
FROM tag
 LIMIT ? OFFSET ?
17:23:06,908 INFO  [sqlalchemy.engine.base.Engine] (20, 0)
17:23:06,908 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:23:06,909 INFO  [root] 200 GET /api/tag (127.0.0.1) 7.38ms
17:28:59,697 INFO  [root] --------------- session started -----------------------
17:28:59,697 INFO  [root] logging facility to "example.log"
17:28:59,697 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:29:03,833 ERROR [root] Could not open static file 'css/console.css'
17:29:03,833 ERROR [root] Could not open static file 'css/ext-all.css'
17:29:03,833 ERROR [root] Could not open static file '/resources/ext-4.0/ext-all-debug.js'
17:29:03,834 ERROR [root] Could not open static file '/resources/app.js'
17:29:03,834 INFO  [root] 200 GET /ui (127.0.0.1) 5.96ms
17:29:04,150 WARNI [root] 404 GET /static/css/ext-all.css (127.0.0.1) 0.45ms
17:29:04,155 WARNI [root] 403 GET /static//resources/app.js (127.0.0.1): /resources/app.js is not in root static directory
17:29:04,156 WARNI [root] 403 GET /static//resources/app.js (127.0.0.1) 0.90ms
17:29:04,157 WARNI [root] 403 GET /static//resources/ext-4.0/ext-all-debug.js (127.0.0.1): /resources/ext-4.0/ext-all-debug.js is not in root static directory
17:29:04,158 WARNI [root] 403 GET /static//resources/ext-4.0/ext-all-debug.js (127.0.0.1) 0.75ms
17:29:04,159 WARNI [root] 404 GET /static/css/console.css (127.0.0.1) 0.36ms
17:29:55,774 INFO  [root] --------------- session started -----------------------
17:29:55,774 INFO  [root] logging facility to "example.log"
17:29:55,774 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:29:57,641 ERROR [root] Could not open static file 'css/console.css'
17:29:57,641 ERROR [root] Could not open static file 'css/ext-all.css'
17:29:57,641 ERROR [root] Could not open static file 'ext-4.0/ext-all-debug.js'
17:29:57,641 ERROR [root] Could not open static file 'app.js'
17:29:57,642 INFO  [root] 200 GET /ui (127.0.0.1) 6.10ms
17:29:57,946 WARNI [root] 404 GET /static/ext-4.0/ext-all-debug.js (127.0.0.1) 0.46ms
17:29:57,950 WARNI [root] 404 GET /static/css/console.css (127.0.0.1) 0.37ms
17:29:57,951 WARNI [root] 404 GET /static/css/ext-all.css (127.0.0.1) 0.34ms
17:29:57,954 WARNI [root] 404 GET /static/app.js (127.0.0.1) 0.48ms
17:41:50,817 INFO  [root] --------------- session started -----------------------
17:41:50,817 INFO  [root] logging facility to "example.log"
17:41:50,818 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:41:54,835 ERROR [root] Could not open static file 'css/ext-all.css'
17:41:54,912 INFO  [root] 200 GET /ui (127.0.0.1) 84.08ms
17:41:55,023 INFO  [root] 200 GET /static/app.js?v=9a8a6 (127.0.0.1) 58.05ms
17:41:55,025 WARNI [root] 404 GET /static/css/ext-all.css (127.0.0.1) 0.39ms
17:41:55,043 INFO  [root] 200 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 16.71ms
17:45:04,827 INFO  [root] --------------- session started -----------------------
17:45:04,827 INFO  [root] logging facility to "example.log"
17:45:04,827 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:45:08,010 ERROR [root] Could not open static file 'css/ext-all.css'
17:45:08,020 INFO  [root] 200 GET /ui (127.0.0.1) 15.47ms
17:45:08,070 WARNI [root] 404 GET /static/css/ext-all.css (127.0.0.1) 0.59ms
17:45:08,083 INFO  [root] 200 GET /static/app.js?v=072d9 (127.0.0.1) 10.23ms
17:45:08,085 INFO  [root] 304 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 0.66ms
17:45:08,801 DEBUG [teg.controller] no filter arguments
17:45:08,801 DEBUG [teg.controller] no sort arguments
17:45:08,802 DEBUG [teg.controller] arguments: {'start': ['0'], '_dc': ['1330091108792'], 'limit': ['25'], 'page': ['1']}
17:45:08,805 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:45:08,806 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page) AS anon_1
17:45:08,807 INFO  [sqlalchemy.engine.base.Engine] ()
17:45:08,808 DEBUG [teg.controller] get page 1, offset 0, limit 25
17:45:08,809 DEBUG [teg.controller] (SELECT page.id, page.content, page.posted, page.title, page.heading 
FROM page
 LIMIT :param_1)
17:45:08,811 INFO  [sqlalchemy.engine.base.Engine] SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page
 LIMIT ? OFFSET ?
17:45:08,811 INFO  [sqlalchemy.engine.base.Engine] (25, 0)
17:45:08,812 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:45:08,813 INFO  [root] 200 GET /api/page?_dc=1330091108792&page=1&start=0&limit=25 (127.0.0.1) 16.55ms
17:45:30,151 INFO  [root] 200 GET /ui (127.0.0.1) 0.42ms
17:45:30,203 INFO  [root] 304 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 0.60ms
17:45:30,206 WARNI [root] 404 GET /static/css/ext-all.css (127.0.0.1) 0.53ms
17:45:30,208 INFO  [root] 200 GET /static/app.js?v=072d9 (127.0.0.1) 0.84ms
17:45:31,055 DEBUG [teg.controller] no filter arguments
17:45:31,056 DEBUG [teg.controller] no sort arguments
17:45:31,056 DEBUG [teg.controller] arguments: {'start': ['0'], '_dc': ['1330091131047'], 'limit': ['25'], 'page': ['1']}
17:45:31,058 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page) AS anon_1
17:45:31,059 INFO  [sqlalchemy.engine.base.Engine] ()
17:45:31,059 DEBUG [teg.controller] get page 1, offset 0, limit 25
17:45:31,060 DEBUG [teg.controller] (SELECT page.id, page.content, page.posted, page.title, page.heading 
FROM page
 LIMIT :param_1)
17:45:31,062 INFO  [sqlalchemy.engine.base.Engine] SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page
 LIMIT ? OFFSET ?
17:45:31,062 INFO  [sqlalchemy.engine.base.Engine] (25, 0)
17:45:31,062 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:45:31,063 INFO  [root] 200 GET /api/page?_dc=1330091131047&page=1&start=0&limit=25 (127.0.0.1) 7.75ms
17:47:21,182 INFO  [root] --------------- session started -----------------------
17:47:21,182 INFO  [root] logging facility to "example.log"
17:47:21,182 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:47:22,705 INFO  [root] 200 GET /ui (127.0.0.1) 62.26ms
17:47:22,769 INFO  [root] 304 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 9.76ms
17:47:22,771 INFO  [root] 200 GET /static/app.js?v=0cec8 (127.0.0.1) 0.78ms
17:47:22,773 INFO  [root] 200 GET /static/resources/css/ext-all.css?v=48681 (127.0.0.1) 2.13ms
17:47:23,591 INFO  [root] 200 GET /static/resources/themes/images/default/form/text-bg.gif (127.0.0.1) 0.67ms
17:47:23,621 INFO  [root] 200 GET /static/resources/themes/images/default/form/exclamation.gif (127.0.0.1) 0.63ms
17:47:23,634 DEBUG [teg.controller] no filter arguments
17:47:23,634 DEBUG [teg.controller] no sort arguments
17:47:23,634 DEBUG [teg.controller] arguments: {'start': ['0'], '_dc': ['1330091243609'], 'limit': ['25'], 'page': ['1']}
17:47:23,637 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:47:23,639 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page) AS anon_1
17:47:23,639 INFO  [sqlalchemy.engine.base.Engine] ()
17:47:23,640 DEBUG [teg.controller] get page 1, offset 0, limit 25
17:47:23,640 DEBUG [teg.controller] (SELECT page.id, page.content, page.posted, page.title, page.heading 
FROM page
 LIMIT :param_1)
17:47:23,642 INFO  [sqlalchemy.engine.base.Engine] SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page
 LIMIT ? OFFSET ?
17:47:23,642 INFO  [sqlalchemy.engine.base.Engine] (25, 0)
17:47:23,643 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:47:23,643 INFO  [root] 200 GET /api/page?_dc=1330091243609&page=1&start=0&limit=25 (127.0.0.1) 20.48ms
17:47:23,644 INFO  [root] 200 GET /static/resources/themes/images/default/dd/drop-no.gif (127.0.0.1) 0.52ms
17:47:23,645 INFO  [root] 200 GET /static/resources/themes/images/default/grid/loading.gif (127.0.0.1) 0.42ms
17:47:23,646 INFO  [root] 200 GET /static/resources/themes/images/default/grid/grid3-hd-btn.gif (127.0.0.1) 0.41ms
17:49:10,218 INFO  [root] 200 GET /ui (127.0.0.1) 0.43ms
17:49:10,618 INFO  [root] 304 GET /static/resources/css/ext-all.css?v=48681 (127.0.0.1) 0.61ms
17:49:10,620 INFO  [root] 304 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 0.78ms
17:49:10,622 INFO  [root] 200 GET /static/app.js?v=0cec8 (127.0.0.1) 0.74ms
17:49:11,197 DEBUG [teg.controller] no filter arguments
17:49:11,197 DEBUG [teg.controller] no sort arguments
17:49:11,198 DEBUG [teg.controller] arguments: {'start': ['0'], '_dc': ['1330091351186'], 'limit': ['25'], 'page': ['1']}
17:49:11,201 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page) AS anon_1
17:49:11,202 INFO  [sqlalchemy.engine.base.Engine] ()
17:49:11,202 DEBUG [teg.controller] get page 1, offset 0, limit 25
17:49:11,203 DEBUG [teg.controller] (SELECT page.id, page.content, page.posted, page.title, page.heading 
FROM page
 LIMIT :param_1)
17:49:11,205 INFO  [sqlalchemy.engine.base.Engine] SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page
 LIMIT ? OFFSET ?
17:49:11,205 INFO  [sqlalchemy.engine.base.Engine] (25, 0)
17:49:11,206 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:49:11,207 INFO  [root] 200 GET /api/page?_dc=1330091351186&page=1&start=0&limit=25 (127.0.0.1) 9.72ms
17:50:14,463 INFO  [root] --------------- session started -----------------------
17:50:14,463 INFO  [root] logging facility to "example.log"
17:50:14,463 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:50:16,490 INFO  [root] 200 GET /ui (127.0.0.1) 15.82ms
17:50:16,555 INFO  [root] 304 GET /static/resources/css/ext-all.css?v=48681 (127.0.0.1) 11.81ms
17:50:16,557 INFO  [root] 304 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 1.07ms
17:50:16,558 INFO  [root] 200 GET /static/app.js?v=3c80b (127.0.0.1) 0.71ms
17:50:17,407 DEBUG [teg.controller] no filter arguments
17:50:17,408 DEBUG [teg.controller] no sort arguments
17:50:17,408 DEBUG [teg.controller] arguments: {'start': ['0'], '_dc': ['1330091417390'], 'limit': ['25'], 'page': ['1']}
17:50:17,412 INFO  [sqlalchemy.engine.base.Engine] BEGIN (implicit)
17:50:17,413 INFO  [sqlalchemy.engine.base.Engine] SELECT count(*) AS count_1 
FROM (SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page) AS anon_1
17:50:17,413 INFO  [sqlalchemy.engine.base.Engine] ()
17:50:17,415 DEBUG [teg.controller] get page 1, offset 0, limit 25
17:50:17,416 DEBUG [teg.controller] (SELECT page.id, page.content, page.posted, page.title, page.heading 
FROM page
 LIMIT :param_1)
17:50:17,418 INFO  [sqlalchemy.engine.base.Engine] SELECT page.id AS page_id, page.content AS page_content, page.posted AS page_posted, page.title AS page_title, page.heading AS page_heading 
FROM page
 LIMIT ? OFFSET ?
17:50:17,418 INFO  [sqlalchemy.engine.base.Engine] (25, 0)
17:50:17,419 DEBUG [teg.controller] 0 objects in page, 0 total objects
17:50:17,420 INFO  [root] 200 GET /api/page?_dc=1330091417390&page=1&start=0&limit=25 (127.0.0.1) 17.00ms
17:52:09,289 INFO  [root] 200 GET /ui (127.0.0.1) 0.63ms
17:52:09,352 WARNI [root] 404 GET /static/ext-4.0/ext-all-debug.js?v=5289e (127.0.0.1) 0.72ms
17:52:09,355 WARNI [root] 404 GET /static/app.js?v=3c80b (127.0.0.1) 0.41ms
17:52:09,357 WARNI [root] 404 GET /static/resources/css/ext-all.css?v=48681 (127.0.0.1) 0.89ms
17:52:15,799 INFO  [root] --------------- session started -----------------------
17:52:15,800 INFO  [root] logging facility to "example.log"
17:52:15,800 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:52:16,895 ERROR [root] Could not open static file 'resources/css/ext-all.css'
17:52:16,895 ERROR [root] Could not open static file 'ext-4.0/ext-all-debug.js'
17:52:16,895 ERROR [root] Could not open static file 'app.js'
17:52:16,896 INFO  [root] 200 GET /ui (127.0.0.1) 7.98ms
17:52:16,950 WARNI [root] 404 GET /static/ext-4.0/ext-all-debug.js (127.0.0.1) 0.48ms
17:52:16,953 WARNI [root] 404 GET /static/resources/css/ext-all.css (127.0.0.1) 0.45ms
17:52:16,954 WARNI [root] 404 GET /static/app.js (127.0.0.1) 0.40ms
17:52:38,227 INFO  [root] --------------- session started -----------------------
17:52:38,227 INFO  [root] logging facility to "example.log"
17:52:38,227 DEBUG [root] logger options: debug=True size=1048576 copies=5 rotating=False
17:52:40,417 INFO  [root] 302 GET / (127.0.0.1) 0.98ms
17:52:40,429 ERROR [root] Could not open static file 'resources/css/ext-all.css'
17:52:40,429 ERROR [root] Could not open static file 'ext-4.0/ext-all-debug.js'
17:52:40,430 ERROR [root] Could not open static file 'app.js'
17:52:40,430 INFO  [root] 304 GET /ui (127.0.0.1) 5.98ms
17:52:40,792 WARNI [root] 404 GET /static/resources/css/ext-all.css (127.0.0.1) 0.57ms
17:52:40,793 WARNI [root] 404 GET /static/ext-4.0/ext-all-debug.js (127.0.0.1) 0.34ms
17:52:40,794 WARNI [root] 404 GET /static/app.js (127.0.0.1) 0.33ms
